target/linux/brcm63xx/patches-4.4/421-BCM63XX-add-led-pin-for-ath9k.patch

```diff
@@ -44,6 +46,6 @@
  
  void pci_enable_ath9k_fixup(unsigned slot, u32 offset,
 -	unsigned endian_check) __init;
-+	unsigned endian_check, int led_pin) __init;
++	unsigned endian_check, int led_pin, bool led_active_high) __init;
  
  #endif /* _PCI_ATH9K_FIXUP */
```

Comment:
Please split this into two commits, one adding active led support, one making hg556a use it. Bonus points for a third commit adding it for dsl-274x1-f1, which needs it as well.